README.md

```diff
@@ -1 +1,109 @@
-# monumenten
\ No newline at end of file
+# Monumenten
+
+Een Python package voor het ophalen van monumentgegevens van Nederlandse overheids-API's. Momenteel is het mogelijk om rijksmonumenten en beschermd gezichten op te halen.
+
+Door middel van dit package is het mogelijk om, indienst gewenst, voor tienduizenden verblijfsobjecten per seconde monumentgegevens op te halen. Er zijn geen API-keys nodig.
+
+## Installatie
+
+`pip install monumenten`
+
+## Gebruik
+
+### Python-native 🐍
+
+```python
+import asyncio
+
+from monumenten import MonumentenClient
+
+async def main():
+    bag_verblijfsobject_ids = [
+        "0599010000360091",
+        "0599010000486642",
+        "0599010000281115",
+    ]
+
+    async with MonumentenClient() as client:
+        result = await client.process_from_list(bag_verblijfsobject_ids)
+        print(result)
+
+if __name__ == "__main__":
+    asyncio.run(main())
+```
+
+```python
+# OUTPUT
+[
+    {
+        'bag_verblijfsobject_id': '0599010000360091',
+        'is_rijksmonument': True,
+        'rijksmonument_nummer': '524327',
+        'rijksmonument_url': 'https://monumenten.nl/monument/524327',
+        'is_beschermd_gezicht': False,
+        'beschermd_gezicht_naam': nan
```

Comment:
Klopt het dat we voor een string `nan` hebben als er geen waarde is?

Reply:
hmm ja dat maakt `pandas` ervan, maar bij de `from_list` functie is het idd mooier om python-native data types te gebruiken, ik ga het aanpassen!

Reply:
opgelost in https://github.com/woonstadrotterdam/monumenten/pull/1/commits/917e1d25a625f7009ab43ed39d31ef776cfd5920